perun-core/src/main/java/cz/metacentrum/perun/core/impl/ConsentsManagerImpl.java

```diff
@@ -13,10 +19,42 @@
  */
 public class ConsentsManagerImpl implements ConsentsManagerImplApi {
 
+	final static Logger log = LoggerFactory.getLogger(ConsentsManagerImpl.class);
+
 	private static JdbcPerunTemplate jdbc;
 
 	public ConsentsManagerImpl(DataSource perunPool) {
 		jdbc = new JdbcPerunTemplate(perunPool);
 		jdbc.setQueryTimeout(BeansUtils.getCoreConfig().getQueryTimeout());
 	}
+
+	@Override
+	public Consent createConsent(PerunSession perunSession, Consent consent) {
+		try {
+			int newId = Utils.getNewId(jdbc, "consents_id_seq");
+
+			jdbc.update("insert into consents(id, user_id, consent_hub_id, consent_status, created_at, created_by, modified_at, modified_by, created_by_uid, modified_by_uid) " +
+					"values (?,?,?,?,"+Compatibility.getSysdate()+",?,"+Compatibility.getSysdate()+",?,?,?)", newId, consent.getUserId(),
+				consent.getConsentHub().getId(), "UNSIGNED", perunSession.getPerunPrincipal().getActor(), perunSession.getPerunPrincipal().getActor(),
```

Comment:
i find it more convenient to use `ConsentStatus.UNSIGNED`, because if we choose to rename it at some point, we won't have to rewrite it at all places where we didn't use the enum value.